ui/admin/app/components/form/host-set/aws/index.hbs

```diff
@@ -50,12 +51,9 @@
     {{/if}}
   </Hds::Form::Textarea::Field>
 
-  <form.input
-    @value={{@model.type}}
-    @label={{t 'form.type.label'}}
-    readonly={{true}}
-    @disabled={{true}}
-  />
+  <InfoField @value={{@model.type}} disabled={{form.disabled}} as |F|>
```

Comment:
Why the new implementation is adding a second `<InfoField>` ? in line 58 we have the previous one.